### ISSUE TITLE
Whatever I save as settings, the client does not auto-connect to the server

### ISSUE THREAD

**Issue by streampiuser** [Stream-Pi/Server#117]
Hello, I am trying to have the stream-pi server/client combo running on the same machine (linux Mint, stream-pi installed through .deb) as a touch-screen jingle player. The client is set -with the toggle switch- to connect to the server on startup. But no matter what, it never does. It also does not connect with the "connect on action click" toggle set. The only way to make it connect is to use the "connect" button, but even then the server panel wants me to pick the client and the profile (of which there is only one each, so there is nothing to choose). If I don't pick those profiles, it will not see the media files the buttons tell it to access.

Is there a way to edit these settings into the xml config file(s) so both the server and client can be hard-forced to run exactly the way I am expecting of them? And if yes, which parts of the xml should be modified?

In short:
expected behaviour: server and client start in the state (connection/client/profile) I saved them in before reboot
seen behaviour: they do not hold the connection/client/profile settings
solution: ?

Thank you very much in advance for your advice.
